pkgs/applications/emulators/dolphin-emu/default.nix

```diff
@@ -93,32 +95,33 @@ stdenv.mkDerivation rec {
     libspng
     libusb1
     libXdmcp
+    lzo
     mbedtls_2
     miniupnpc
     minizip-ng
     openal
     pugixml
     qtbase
+    qtsvg
     sfml
-    soundtouch
```

Comment:
Dolphin was using the vendored soundtouch regardless ([intentionally](https://github.com/dolphin-emu/dolphin/blob/14a60761983abb0364161eb69d17376d512c0648/CMakeLists.txt#L698-L701)).